README.md

```diff
@@ -32,3 +32,27 @@ To run tests:
 ## CI
 
 This plugin is currently built on [ci.jenkins.io](https://ci.jenkins.io/job/Plugins/job/jms-messaging-plugin/).
+
+## Jenkinsfile Build Trigger Example
+
+This example shows triggering your pipeline from Fedmsg with the org.fedoraproject.prod.buildsys.build.state.change topic
+
+````
+pipelineTriggers(
```

Comment:
This is in the context of the properties step in a Jenkinsfile?

Reply:
That is correct. I can put that if you want; pipelineTriggers going in properties was found by me on google, but syntax for jms triggering was not. But yeah, do you think it makes it more complete? If so, I'll add it.

Reply:
I would add it to be more complete.

By the way, have you tried the ```Pipeline Syntax``` page?

Reply:
Never thought to look under properties for the trigger